PaperCut.ps1

```diff
@@ -251,7 +251,7 @@ function Idm-UsersRead {
                     $i = 0
                     foreach($user in $userList) {
                         $i++
-                        Log info "Processing user $($i) of $($userList.count)"
+                        #Log info "Processing user $($i) of $($userList.count)"
```

Comment:
We should avoid leaving commented out code in the connector. If we aren't using it, we should remove it.